### PR TITLE
Code & comments while initiating Gilli into Void

### PR DESCRIPTION
**Pull request by Kazark**
Not much behavior here... mostly a few thoughts expressed as code and comments.

Now may there be two developers on Void!

### CODE REVIEW

File: Void/Void.Core/Commands.fs [hunk 1]
```diff
@@ -3,10 +3,13 @@
 [<RequireQualifiedAccess>]
 type Command =
     | Noop
+    | PublishEvent of Event
+    //
```
Comment (mjgpy3) [inline Void/Void.Core/Commands.fs:7]: Dead comment

File: Void/Void.Core/Commands.fs [hunk 2]
```diff
@@ -3,10 +3,13 @@
 [<RequireQualifiedAccess>]
 type Command =
     | Noop
+    | PublishEvent of Event
+    //
+    | ChangeToMode of Mode
```
Comment (mjgpy3) [inline Void/Void.Core/Commands.fs:8]: What would you think about the name `NextNode`? `ChangeToNode` sounds like an action not a type constructor.

Comment (Kazark) [inline Void/Void.Core/Commands.fs:8]: @mjgpy3 Node? We're talking "Mode", not "Node"... I'm a little confused here.

File: Void/Void.Core/Commands.fs (hunk 1 above)
Comment (Kazark) [inline Void/Void.Core/Commands.fs:7]: Intended to be a separator... but maybe a bad idea

File: Void/Void.Core/Commands.fs (hunk 2 above)
Comment (mjgpy3) [inline Void/Void.Core/Commands.fs:8]: @Kazark yep sorry, consistent mistypes, wow.

Comment (Kazark) [inline Void/Void.Core/Commands.fs:8]: So you want to call it `NextMode`? This a command, so it should sound like an action, in my opinion...

Comment (mjgpy3) [inline Void/Void.Core/Commands.fs:8]: I follow now.

Comment (Kazark) [inline Void/Void.Core/Commands.fs:8]: Okay. So long as you are good with that. I don't want to shut you down on that, just make sure you understand my reasoning.

Comment (mjgpy3) [inline Void/Void.Core/Commands.fs:8]: Absolutely. I was blindly reciting the "entities are things and functions are actions rule."

**Comment by mjgpy3** [comment]
Seems to be already merged. Close?

**Comment by Kazark** [comment]
Huh? Shouldn't be merged yet unless I screwed something up... though I may have screwed somethign up.

**Pull request merged**